src/Stratis.Bitcoin/Consensus/ConsensusManager.cs

```diff
@@ -27,11 +28,11 @@ public class ConsensusManager : IConsensusManager
         /// Maximum memory in bytes that can be taken by the blocks that were downloaded but
         /// not yet validated or included to the consensus chain.
         /// </summary>
-        private long maxUnconsumedBlocksDataBytes = 200 * 1024 * 1024;
+        public long MaxUnconsumedBlocksDataBytes { get; private set; }
```

Comment:
@mikedennis this can still have an access modifier of private or protected  as it is not being used outside of the component.